api/.env.testing

```diff
@@ -105,3 +105,4 @@ ZENDESK_SELL_API_URL=https://api.getbase.com
 ZENDESK_SELL_BACKEND=pcapi.core.external.zendesk_sell_backends.logger.LoggerBackend
 BIG_QUERY_NOTIFICATIONS_TABLE_BASENAME="passculture-data-ehp.backend_dev"
 SESSION_COOKIE_SAMESITE="None"
+SECRET_ENCRYPTION_KEY=n44qmlL99bsthLkP6pTCqSSgxSQxYeX08RqDsBiDyDQ=
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏ça ne devrait pas être dans les secrets de GCP plutôt ?

Reply:
ça l’est, et pas la même

Reply:
C’est juste que dans les pipelines pour le déploiement du coup on en a besoin `install_local_providers` dans `clean_database` on n’arrive pas à y accéder